front/app/src/js/components/game/Engine/_GameSocketIO.js

```diff
@@ -9,7 +9,7 @@ import {PlayerLocation} from '../Scene/PlayerLocation.js';
 import {sleep} from '../sleep.js';
```

Comment:
This file is full of refactoring so I highlighted the important changes with comments